packages/terra-tag/src/Tag.scss

```diff
@@ -0,0 +1,72 @@
+
+:local {
+  .tag {
+    align-items: center;
+    background-color: var(--terra-tag-background-color, rgba(28, 31, 33, 0.04));
+    border-color: var(--terra-tag-border-color, rgba(28, 31, 33, 0.1));
+    border-radius: 0.2142rem;
+    border-style: var(--terra-tag-border-style, solid);
+    border-width: 0.07142rem;
+    box-shadow: var(--terra-tag-box-shadow, none);
+    color: var(--terra-tag-color, #434a4d);
+    display: inline-flex;
+    font-size: var(--terra-tag-font-size, 1rem);
+    font-weight: var(--terra-tag-font-weight, normal);
+    line-height: var(--terra-tag-line-height, 1.14285);
+    margin-bottom: var(--terra-tag-margin-bottom, 0.357rem);
+    margin-left: var(--terra-tag-margin-left);
+    margin-right: var(--terra-tag-margin-right, 0.714rem);
+    margin-top: var(--terra-tag-margin-top, 0.357rem);
+    padding-bottom: var(--terra-tag-padding-bottom, 0.1428rem);
+    padding-left: var(--terra-tag-padding-left, 0.5rem);
+    padding-right: var(--terra-tag-padding-right, 0.5rem);
+    padding-top: var(--terra-tag-padding-top, 0.1428rem);
+    text-align: left;
+    text-decoration: var(--terra-tag-text-decoration, none);
+    text-transform: none;
+    touch-action: manipulation; // Enable fast tap interaction in webkit browsers; see https://webkit.org/blog/5610/more-responsive-tapping-on-ios/
+    user-select: none; // Prevent text selection on tags on mobile and desktop.
+    white-space: normal;
+
+    // Remove the inner border and padding in Firefox and IE.
+    &::-moz-focus-inner {
+      border: 0;
+      padding: 0;
+    }
+
+    &:focus {
+      outline: none;
+    }
+  }
+
+  .is-interactive {
+    color: var(--terra-tag-interactive-color, #0059a8);
```

Comment:
If it's important to have a different color for interactive tabs through theming, shouldn't we do the same for the background?

Reply:
Added it [here](https://github.com/cerner/terra-core/pull/1306/commits/b04c1661bc417ddbf31b7da23c28d95ec857e1df).